client/gutenberg/extensions/tiled-gallery/layout-square.jsx

```diff
@@ -1,3 +1,5 @@
+/** @format */
```

Comment:
If we're reorganizing, I'm a big fan of one component per file. Could we split out the `TiledGallerySquareGroup`?